src/mscorlib/shared/System/StringComparer.cs

```diff
@@ -163,32 +172,50 @@ public int GetHashCode(object obj)
 
     [Serializable]
     [System.Runtime.CompilerServices.TypeForwardedFrom("mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089")]
-    public sealed class CultureAwareComparer : StringComparer
+    public sealed class CultureAwareComparer : StringComparer, ISerializable
     {
+        private const CompareOptions ValidCompareMaskOffFlags = ~(CompareOptions.IgnoreCase | CompareOptions.IgnoreSymbols | CompareOptions.IgnoreNonSpace | CompareOptions.IgnoreWidth | CompareOptions.IgnoreKanaType | CompareOptions.StringSort);
+
         private readonly CompareInfo _compareInfo; // Do not rename (binary serialization)
-        private readonly bool _ignoreCase; // Do not rename (binary serialization)
+        private CompareOptions _options;
```

Comment:
that one should be readonly

Reply:
I just checked with net471, they named the field the same. lucky us.
https://referencesource.microsoft.com/#mscorlib/system/stringcomparer.cs,140